### PR TITLE
remove metaprogramming from ApplicationController initialization. Hel…

### PR DESCRIPTION
**Pull request by mejackreed**
…per modules can be explicitly included in controllers if needed (probably a smell though)

This was initially added in the infamous Padrino conversion commit 50cf101286104310f970041b867ddb38aee69500

### CODE REVIEW

**Comment by mejackreed** [comment]
I think the coverage drop here is a result of unexpected craziness.

File: app/controllers/registration_controller.rb [hunk 1]
```diff
@@ -9,7 +10,7 @@ def tracksheet
     name     = params[:name] || 'tracksheet'
     sequence = params[:sequence] || 1
     response['content-disposition'] = "attachment; filename=#{name}-#{sequence}.pdf"
-    pdf = help.generate_tracking_pdf(druids)
+    pdf = generate_tracking_pdf(druids)
```
Comment (cbeer) [inline app/controllers/registration_controller.rb]: Does this method really belong in a helper, or maybe it should be its own class/model?

Comment (mejackreed) [inline app/controllers/registration_controller.rb]: Oh.. totally agree here. Just /shrug.... so much of that

Comment (mejackreed) [inline app/controllers/registration_controller.rb]: And... the tests aren't quite there... https://github.com/sul-dlss/argo/blob/develop/spec/helpers/registration_helper_spec.rb

Comment (mejackreed) [inline app/controllers/registration_controller.rb]: Created #348

**Pull request merged**